examples/configuration/UsingSSHToConfigureNameResolution.ipynb

```diff
@@ -50,23 +52,72 @@
     "# Set the subnet\n",
     "export NETWORK_SUBNET_CIDR=\"192.168.100.0/24\"\n",
     "\n",
-    "# Set the fixed IP assigned to the server\n",
+    "## Server information\n",
+    "export SERVER_NAME='myServer'\n",
+    "# Server's fixed IP inside our private network \n",
     "export SERVER_IP='192.168.100.10'\n",
     "\n",
-    "#Set the number of clients to create\n",
+    "## Client information\n",
+    "export CLIENT_NAME='myClient'\n",
+    "#Set the number of clients to create. \n",
+    "# First client will be called $CLIENT_NAME-1\n",
+    "# Last client will be called $CLIENT_NAME-$CLIENT_COUNT\n",
     "export CLIENT_COUNT=2\n",
+    "# Client's private network IPs will be automatically generated (think DHCP)\n",
     "\n",
     "# Add Chameleon custom blazar client\n",
     "#sudo pip install -e git+https://github.com/ChameleonCloud/python-blazarclient.git@allocatable-vlans#egg=python-blazarclient\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Make SSH keypair available in openstack\n",
+    "Before we create servers and reservations and networks, we do need to have the ssh keypair we want to use in openstack. So we ask openstack"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "openstack keypair list"
```

Comment:
This may be helpful to use, it is available by default in the Jupyter environment: https://github.com/ChameleonCloud/jupyterhub-deploy/blob/master/singleuser/bashrc.d/chameleon_functions.sh#L83 - it may also make the next step unnecessary.

Reply:
Let's do it in a future version; we ran out of time here.